src/App/Pages/Accounts/RegisterPage.xaml

```diff
@@ -59,7 +60,8 @@
                         IsTextPredictionEnabled="False"
                         IsPassword="{Binding ShowPassword, Converter={StaticResource inverseBool}}"
                         Grid.Row="1"
-                        Grid.Column="0" />
+                        Grid.Column="0"
+                        AutomationId="MasterPasswordField"/>
```

Comment:
🤔 some of these have a suffix related to the item type, most don't. Can we define a standard so our devs can follow it too? It should:

1. Define a name for each element type
    * Button -> Button
    * Switch, IconButton -> Toggle
    * Label -> Label
    * Entry, MonoEntry -> Input
    * ...
1. Define if we prefix or suffix those names. `ButtonLoginWithDevice` vs `LoginWithDeviceButton`
1. Be relevant for you and your team, you don't need to follow the same code element names. They should still make sense even if the implementation changes.
    
That list is just an example and I'm not trying to be prescriptive about nomenclature, feel free to define what works better for you / your team!

Reply:
Hey! Yes, this is my fault! I planned to use the same standard on all the IDs I added, but I didn't. It'll be fixed on my next commit 💪 

Also, I'll explain the standard on the document I mentioned yesterday 🙌